shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -952,4 +958,74 @@ void PlatformView::HandleFlutterPlatformViewsChannelPlatformMessage(
   }
 }
 
+void PlatformView::HandleFuchsiaShaderWarmupChannelPlatformMessage(
+    fml::RefPtr<flutter::PlatformMessage> message) {
+  FML_DCHECK(message->channel() == kFuchsiaShaderWarmupChannel);
+
+  if (on_shader_warmup_) {
```

Comment:
Refactor to do an early return:

if (!on_shader_warmup_) {
    message->response()->Complete(....blahblah);
    return;
}

// json parsing

Reply:
done